Assets/MixedRealityToolkit.Extensions/Webrtc/Scripts/Webrtc.cs

```diff
@@ -0,0 +1,369 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Events;
+using Microsoft.MixedReality.Toolkit.Extensions.Webrtc.Marshalling;
+
+#if UNITY_WSA && !UNITY_EDITOR
+using Windows.UI.Core;
+using Windows.Foundation;
+using Windows.Media.Core;
+using Windows.Media.Capture;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Core;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Webrtc
```

Comment:
>Webrtc [](start = 52, length = 6)

nit: given RTC is an acronym it may be worth capitalizing. After this initial review is checked in it will be difficult to change.